package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.0.2",
   "description": "",
   "main": "src/index.js",
+  "engines": {
+    "node": ">=9.10.x"
```

Comment:
Can we make this `8.x.x`? That's the release I'm on.

Reply:
So I’m not sure if you know this but this script requires three very specific versions of node. (I don’t remember the three though) so 8.x.x might not work

Reply:
Greater than 9.10 is one of them

Reply:
Gotcha. What feature requires greater than 9.10? I was able to run master fine under 8.13

Reply:
I’m not too sure. My process was got clone and yarn, where the message popped up. So I’m not too sure which feature(s) requires which version(s).

Reply:
kk. Let's stick with 8.x.x for now because it's an LTS and tends to be what most people use. If we find features that require future node versions, we should try and modify those features so they can run on lower versions of node.

Reply:
sooooo The NVM issue I ran into that prompted me to create this PR no longer exists. I'm happy to ignore this PR for now if you don't deem it necessary.